### ISSUE TITLE
Warnings because of optional inputs

### ISSUE THREAD

**Issue by goelKriti** [Macaulay2/M2#2516]
Hello,
I am getting the following warnings while installing the package
![Screenshot (154)](https://user-images.githubusercontent.com/65620879/171319138-2aae3ef1-1003-43ab-a455-78c45ef15e2b.png)
They seem to appear because VariableBaseName is one of the optional inputs in a function of the package. Also, the function tensor appearing in the warning signs is just one of the functions which also have VariableBaseName as an optional input. However, this function doesn't appear in the package which is getting installed.
![Screenshot (155)](https://user-images.githubusercontent.com/65620879/171319140-1a60ee6e-61fa-4eec-b834-2614d5186fb8.png)

**Comment by DanGrayson** [comment]
Yes, there doesn't seem to be any reason to display all the other documentation nodes for functions accepting the same optional argument name on this node.  Thanks!  I'll take care of it.

**Comment by DanGrayson** [comment]
@mahrud -- I don't understand your code below.  This function doesn't seem to use the second argument, `S'`.  How would you fix things so the "Functions with optional argument named ..." section is displayed only when the symbol itself is documented?
```m2
documentationValue(Symbol, Symbol) := (S, S') -> (
    -- return links to all other methods with option name Strategy
    initializeReverseOptionTable();
    -- functions that take S as option
    opts := if reverseOptionTable#?S then keys reverseOptionTable#S else {};
    reverseOptionTable = null;
    -- TODO: should we only list methods with the same option name in
    -- the same package? select for package f === package currentHelpTag
    a := smenu apply(select(opts, f -> isDocumentableMethod f), f -> [f, S]);
    if #a > 0 then DIV { -- "class" => "waystouse", -- we want this one to be larger
	 SUBSECTION {"Functions with optional argument named ", TT toString S, " :"}, a})
```
